src/mongo/client/dbclientinterface.h

```diff
@@ -897,7 +897,13 @@ namespace mongo {
 
            returns true if successful
         */
-        bool copyDatabase(const std::string &fromdb, const std::string &todb, const std::string &fromhost = "", BSONObj *info = 0);
+        bool copyDatabase(const std::string &fromdb,
+                          const std::string &todb,
+                          const std::string &fromhost = "",
+                          const std::string &mechanism = "DEFAULT",
```

Comment:
The [DRIVERS ticket](https://jira.mongodb.org/browse/DRIVERS-188) for this was pretty explicit that the default value be either "DEFAULT" or another special value, so though I agree it would be nice to simplify it, I don't think it's an option.